### ISSUE TITLE
Facebook thinks fbchat logs into a phishing website!

### ISSUE THREAD

**Issue by S3llout** [carpedm20/fbchat#343]
Recently logged into my second facebook account to check its health to be greeted with text saying that it's been logged into from a phishing website.

I don't really know what to say because now its suspended...

**Comment by madsmtm** [comment]
Yeah, I've been having a lot of trouble with these things as well. Did you provide a phone number for the account? Usually that makes Facebook trust the account a little more.

**Comment by madsmtm** [comment]
Most probably the repeated logging in / out. You can also try saving the session to file (see [`Client.getSession`](https://fbchat.readthedocs.io/en/master/api.html#fbchat.Client.getSession) and [`Client.__init__.session_cookies`](https://fbchat.readthedocs.io/en/master/api.html#fbchat.Client)). AFAICT Facebook doesn't have much of a problem with us not logging in from a browser.

**Comment by emensee** [comment]
I had this same problem. Logging in with my uid and password seems to work. enter it as a int not a string, ie no " ". Also I put in max_tries=1 to reduce the logging attempts. Hope this helps someone.

**Comment by emensee** [comment]
I had this same problem. Logging in with my uid and password seems to work. enter it as a int not a string, ie no " ". Also I put in max_tries=1 to reduce the logging attempts. Hope this helps someone.
